spec/models/piece_spec.rb

```diff
@@ -50,4 +54,124 @@
       king.errors.clear
     end
   end
+
+  describe 'is_obstructed?' do
+    before :all do
+      # @user = FactoryGirl.create(:user)
```

Comment:
should just delete out any commented code to keep things clean. we can always add it in later if we need to use it